rules.txt

```diff
@@ -666,6 +677,7 @@
 /media/20211210131210/2021_CUA_Scorecard_print.pdf  301  /documents/cua-scorecards/
 /media/20230517102700/HCW-guidance-updated-_5.9.23-.pdf  301  /programs/coronavirus-disease-2019-covid-19/guidance/guidance-documents/vaccine-and-masking-requirements-for-healthcare-workers/ 
 /media/20240104103841/Pensions-investment-policy-20231026.pdf   301  /media/20240220161559/Pensions-IPS-202402.pdf
+/media/20170209150802/CertifiedCopy16017601-1.pdf 301 https://codelibrary.amlegal.com/codes/philadelphia/latest/philadelphia_pa/0-0-0-300626
```

Comment:
Don't forget we need to make sure to update the cloudfront behavior in the /media path to use the latest router lambda arn after we push this to prod!